kernel/interrupts/src/aarch64/mod.rs

```diff
@@ -75,14 +85,8 @@ pub struct ExceptionContext {
     esr_el1: EsrEL1,
 }
 
-#[derive(Debug, PartialEq, Eq)]
-#[repr(C)]
-pub enum EoiBehaviour {
-    CallerMustSignalEoi,
-    HandlerHasSignaledEoi,
-}
-
-type HandlerFunc = extern "C" fn(&ExceptionContext) -> EoiBehaviour;
+pub type HandlerFunc = extern "C" fn(&InterruptStackFrame) -> EoiBehaviour;
```

Comment:
i vote to rename this `InterruptHandler` for clarity. Applies to both archs.

Reply:
I agree with this name.